dask_glm/regularizers.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+
+
+class L2(object):
```

Comment:
Yea, that makes sense; I don't have a strong opinion on this, other than to say the current organization is easy (for me personally) to manage.